saturn-r/postBuild

```diff
@@ -0,0 +1,48 @@
+# https://cran.rstudio.com/bin/linux/debian/
+sudo apt-key adv --keyserver keys.gnupg.net --recv-key 'E19F5F87128899B192B1A2C2AD5F960A256A04AF'
+
+sudo apt-get update
+sudo apt-get install -y \
+    --no-install-recommends \
+        software-properties-common
+
+sudo su -c \
+    "echo 'deb http://cloud.r-project.org/bin/linux/debian buster-cran40/' >>/etc/apt/sources.list"
+
+sudo apt update
+sudo apt install -y -t buster-cran40 \
+    r-base \
+    r-base-dev 
+
+sudo apt-get update
+sudo apt-get install -y \
+    jupyter-client \
+    devscripts \
+    libxml2 \
+    libxml2-dev \
+    libcurl4-openssl-dev \
+    libssl-dev \
+    libzmq3-dev \
+    qpdf
+
+# make it possible to `install.packages()` - write permissions required.
+sudo chown 1000:1000 -R $R_LIBS
+sudo chmod 777 -R $R_LIBS
+
+# install other packages
+sudo Rscript -e "install.packages(c( \
```

Comment:
Okay, here is the list of packages we should have installed in this. Note that I am okay with (but don't _love_) swapping tidyverse for the 8 core tidyverse packages (ggplot2, dplyr, tidyr, readr, purrr, tibble, stringr, forcats):

* tidyverse
* data.table
* lightgbm
* Rcpp
* lubridate
* remotes
* devtools
* reticulate
* xgboost
* tidymodels
* IRkernel
* tensorflow
* keras

I think this is a nice amount of packages that people should be able to do basic stuff without being bloated. I would like us to come to a consensus on this and publishing it somewhere where people can see it rather than have it kind of be behind the scenes. People should know what they'll need to install themselves or not

Reply:
I think tidyverse is a bit bloated, so I'm strongly in favor of using explicitly just the libraries we want and keeping it nimble. Also, this may just be my ignorance, but do people use tibble a whole lot? Or is that more needed because of interdependencies with other libraries that force people into tibble format?